Generator/BotBuilderVSIX-V4/ProjectTemplates/Bot Framework/wwwroot/default.htm

```diff
@@ -8,7 +8,7 @@
 <body style="font-family:'Segoe UI'">
     <h1>EchoBot sample using ASP.Net Core 2</h1>
     <p>Describe your bot here and your terms of use etc.</p>
-    <p>To debug you bot using the Bot Framework Emulator, paste this URL into the Emulator window:</p>
+    <p>To debug your bot using the Bot Framework Emulator, paste this URL into the <strong>New Bot / Endpoint URL textbox:</stong></p>
```

Comment:
Typo here:
`</stong>` is wrong, and is should be `</strong> `

Reply:
Fixed